app/controllers/api/comments_controller.rb

```diff
@@ -1,7 +1,7 @@
 class Api::CommentsController < Api::ApplicationController
 
   def create
-    if debate
+    if current_debate
```

Comment:
you use current_debate more then once. you should add caching because each time it executes a db query (in TokenRequester)